src/sentry/tasks/post_process.py

```diff
@@ -1559,6 +1573,7 @@ def check_if_flags_sent(job: PostProcessJob) -> None:
         process_commits,
         handle_owner_assignment,
         handle_auto_assignment,
+        kick_off_seer_automation,
```

Comment:
Do any later steps here rely on the `last_run` or `fixability_score` being set? For example, are there any plans to have alert rules send when an issue has a certain fixability score? Or is it just for use in the issue feed and ui?

Reply:
Those fields are just for use in the issue feed and the UI, yes. The idea here is just that we kick off issue summary on every issue. At some point in the background, issue summary will complete and set the `fixability_score` and trigger Autofix. But that can complete in its own time and nothing else is waiting for that.

Reply:
(Slack alerts will try to fetch the issue summary as they do already, but they don't care about those fields)